sdk/src/client/node_manager/syncing.rs

```diff
@@ -73,13 +73,14 @@ impl ClientInner {
             match crate::client::Client::get_node_info(node.url.as_ref(), node.auth.clone()).await {
                 Ok(info) => {
                     if info.status.is_healthy || ignore_node_health {
-                        match network_nodes.get_mut(info.protocol.network_name()) {
+                        // TODO: is it okay to index like this?
```

Comment:
No it's not, the correct one won't always be at index 0, it depends on the start_epoch. See my thread on `eng-iota-2-alignment`

Reply:
figures